Lib/src/packets/smallEvents/SmallEventLotteryPacket.ts

```diff
@@ -0,0 +1,27 @@
+import {SmallEventPacket} from "./SmallEventPacket";
+
+export class SmallEventLotteryNoAnswerPacket extends SmallEventPacket {}
+
+export class SmallEventLotteryPoorPacket extends SmallEventPacket {}
+
+export class SmallEventLotteryWinPacket extends SmallEventPacket {
+	lostTime!: number;
+
+	money?: number;
+
+	points?: number;
+
+	guildXp?: number;
+
+	xp?: number;
+
+	level!: string;
```

Comment:
du coup j'ai un peu plus compris pourquoi t'as mis des chaines en dur, et je vois que t'as typé pour faire en sorte d'autoriser uniquement les 3 bonnes chaines presque partout, manque que ici 😉
```suggestion
	level!: "easy" | "medium" | "hard";
```